src/main/java/game/stores/ItemStore.java

```diff
@@ -0,0 +1,20 @@
+package game.stores;
+
+import game.Item;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ItemStore {
+
+    private List<String> startItems;
+    private List<Item> itemList;
+
+    public Map<String, Item> toStringItemMap() {
+        Map<String, Item> result = new HashMap<>();
+        itemList.forEach(item -> result.put(item.getName(), item));
+
+        return result;
```

Comment:
Nice use of hashmaps to place the action, scene and item objects in. Especially if you are going to use multiple ones. It will be easier to find and use them using the key.